include/rijndael.h

```diff
@@ -18,9 +18,10 @@
 #ifndef RIJNDAEL_H
 #define RIJNDAEL_H
 
+typedef unsigned char   u8;
+typedef unsigned int    u32;
```

Comment:
Unsigned char might be 8 bits, but unsigned int will not be 32 bits on all platforms.

You'll need better typedefs, stdint.h should provide a nice `uint32_t`.